templates/search/blocks/search_filters.html

```diff
@@ -67,51 +78,63 @@ <h2 class="govuk-accordion__section-heading tna-heading search-filters__heading"
                     <div class="search-filters__form-block">
                         <h3 class="tna-heading sr-only">Edit filters</h3>
 
-                        {% if form.group.value == 'tna' or form.group.value == 'digitised' or form.group.value == 'nonTna' %}
+                        {% if form.group.value == 'tna' or form.group.value == 'digitised' %}
                         <div class="search-filters__accordion-section">
-                            <fieldset role="group" aria-describedby="from-date">
+                            <fieldset role="group" aria-describedby="opening-start-date">
                                 <legend>
-                                    <h4 class="tna-heading search-filters__accordion-section-heading" id="record_covering_date">Dates</h4>
+                                    <h4 class="tna-heading-s search-filters__accordion-section-heading" id="record_opening_date">Record opening date</h4>
```

Comment:
```suggestion
                                    <h4 class="tna-heading search-filters__accordion-section-heading" id="record_covering_date">Dates</h4>
```
Same for Covering dates/Dates. Restore previous values.